src/SFA.DAS.FAA.Api/Controllers/SavedSearchesController.cs

```diff
@@ -92,4 +93,22 @@ public async Task<IActionResult> GetSavedSearchCount([FromQuery] Guid candidateI
 
         return Ok(new GetSavedSearchCountResponse(candidateId, result));
     }
+
+    [HttpDelete]
+    [Route("{id:guid}")]
+    [ProducesResponseType<GetSavedSearchCountResponse>((int)HttpStatusCode.NoContent)]
```

Comment:
Response needs removing.

Reply:
I could have been more descriptive there (it's early Monday morning): Just need to remove the GetSavedSearchCountResponse from the Produces attribute.

Reply:
Good spot! Updated and pushed